fabmukunzi/posinnove-dashboard
# feat(profile): user profile page

# This PR is here for:
- Profile page
## It must contain
- profile data
- profile social media links
- experience
- Working experience

## How to test this:
- Run `git clone https://github.com/fabmukunzi/posinnove-dashboard`
- Run `git checkout -b feat-user-profile`
- Run `git pull origin feat-user-profile`
- Run `yarn install` to install packages
- Run `yarn dev` to start the app
Open app in your fev browser
check if those features are present

![image](https://github.com/user-attachments/assets/e439524f-477e-42c4-9989-eb799c4f2013)


![image](https://github.com/user-attachments/assets/9777d88a-badd-47bc-8323-b5e8f76b859f)
